cask.el

```diff
@@ -730,7 +730,7 @@ If DEEP is t, all dependencies recursively will be returned."
            (-map
             (lambda (dependency)
               (list (cask-dependency-name dependency)
-                    (cask-dependency-version dependency)))
+                    (or (cask-dependency-version dependency) "0")))
```

Comment:
Idk.  I don't think that there's a “correct” way.  @lxsameer correctly pointed out that this is actually how Emacs itself behaves (it adds "0" for missing versions), so we can just as well do that in Cask, too.